.gitlab-ci.yml

```diff
@@ -725,11 +725,12 @@ syntax-check:
     - ansible-playbook -i inventory/local-tests.cfg -u root -e ansible_ssh_user=root  -b --become-user=root extra_playbooks/upgrade-only-k8s.yml -vvv  --syntax-check
   except: ['triggers', 'master']
 
-yamllint:
+lint:
   <<: *job
   stage: unit-tests
   script:
     - yamllint roles
+    - pre-commit run -a
```

Comment:
Why run ansible-lint and yamllint in the same step instead of parallel? It might be useful information to know a PR is failing the ansible-lint part, but passes yamllint and vice versa.

Reply:
In fact pre-commit can also take care of running yamllint by itself and it would be faster and with easier to read feedback. Still, this can be implemented in a separated change.